crowd_anki/config/config_dialog.py

```diff
@@ -0,0 +1,75 @@
+"""
+Config dialog entry point.
+This module will load the config initially.
+Write actions will be triggered here when the user clicks the dialog "Save All" button.
+"""
+
+from aqt.qt import *
+
+from .config_ui import Ui_Dialog as ConfigUI
+from .config_settings import ConfigSettings
+
+from PyQt5 import QtCore, QtGui, QtWidgets
+
+class ConfigDialog(QDialog):
+    def __init__(self, parent=None):
+        super().__init__()
+        self.parent = parent
+        self.config = ConfigSettings()
+        self.form = ConfigUI()
+        self.form.setupUi(self)
+        self.ui_initial_setup()
+
+    def accept(self):
+        invalid_values = self.config.find_invalid_config_values()
```

Comment:
this one is more of an opinion, but something like `self.config.validate()` that throws an exception with problem description + try block here that'd display that error if it's thrown would probably work better here

Reply:
Definitely opinion, but well received :grin: I try and avoid try catch blocks when I can. This method works fine for now, and I expect when there are other values in the config then we may wish to run some other arbitrary code without interrupting the flow. If you don't mind I'll leave it the way it is for now.

Reply:
well presumably you won't be interrupting the flow, as you'd wrap the validate in the try 😛.